packages/feedback/src/core/components/Actor.css.ts

```diff
@@ -42,7 +42,8 @@ export function createActorStyles(): HTMLStyleElement {
 }
 
 .widget__actor:hover {
-  background-color: var(--trigger-background-hover);
+  background: var(--actor-hover-background, var(--background));
+  filter: var(--interactive-filter);
```

Comment:
ooo what does filter do?

Reply:
it creates the hover effect, the value is set to be `brightness(95%)`.

so people only need to set one color